src/UIState/SeePIDConstants.cpp

```diff
@@ -10,7 +10,7 @@
 #include "TC_util.h"
 
 void SeePIDConstants::loop() {
-  int elapsedSeconds = (millis() - startTime) / 1000;
+  int elapsedSeconds = ((millis() - startTime) / 1000) % 3;
   switch (elapsedSeconds) {
     case 0:
       loadKp(0);
```

Comment:
What happened to the blank line?

Reply:
What we showed today was three seconds per "frame".